lib/inc/drogon/Cookie.h

```diff
@@ -84,51 +79,31 @@ class DROGON_EXPORT Cookie
     /**
      * @brief Set the domain of the cookie.
      */
-    void setDomain(const std::string &domain)
-    {
-        domain_ = domain;
-    }
-
-    void setDomain(std::string &&domain)
+    void setDomain(std::string domain)
     {
         domain_ = std::move(domain);
     }
 
     /**
      * @brief Set the path of the cookie.
      */
-    void setPath(const std::string &path)
-    {
-        path_ = path;
-    }
-
-    void setPath(std::string &&path)
+    void setPath(std::string path)
     {
         path_ = std::move(path);
     }
 
     /**
      * @brief Set the key of the cookie.
      */
-    void setKey(const std::string &key)
-    {
-        key_ = key;
-    }
-
-    void setKey(std::string &&key)
+    void setKey(std::string key)
```

Comment:
I tried doing this and it leads to ambiguity errors:
```sh
error: call of overloaded ‘setPath(const char [2])’ is ambiguous
  759 |                 sessionid.setPath("/");
      |                 ~~~~~~~~~~~~~~~~~^~~~~
```

Because it doesn't know whether to construct an rvalue string or a string by value

Reply:
Shall I revert this one too?